tests/db_engine_specs/postgres_tests.py

```diff
@@ -274,7 +298,15 @@ def test_extract_errors(self):
                     "and can't be reached on port 12345"
```

Comment:
I think example.com should be "example.com" for some reason I can't add this as a commit suggestion.

Reply:
+1